platform/mbed_rtc_time.h

```diff
@@ -89,6 +89,30 @@ void set_time(time_t t);
  */
 void attach_rtc(time_t (*read_rtc)(void), void (*write_rtc)(time_t), void (*init_rtc)(void), int (*isenabled_rtc)(void));
 
+/** Standard lib retarget, get time since Epoch
+ *
+ * @param timeval   Structure containing time_t secondsa and suseconds_t microseconds. Due to
+ *                  separate target specific RTC implementations only the seconds component is used.
+ *                  It is recommended to use the Mbed OS Time API.
```

Comment:
What Mbed OS Time API? I don't think we've got anything better than this.

Reply:
This was more a reference to recommending users use time/set_time and point them to the time API page in the docs as its a cleaner implementation (just returns seconds instead of using the timeval struct pointer with a usec member thats unused at the moment):
https://os.mbed.com/docs/latest/apis/time.html

This was mainly intended to redirect a user trying to use the stdlib time function to the Mbed variation.

Reply:
> This was more a reference to recommending users use time/set_time

Oh, I see. I guessed that meant "use `Timer`/`Timeout` etc because they can give you fractions of a second" (despite not being wallclock time). 

If you mean "don't bother using this because the microseconds aren't filled in - just use `time()`", that's a bit pessimistic. The implementation can be improved.